chemprop/featurizers/__init__.py

```diff
@@ -1,22 +1,22 @@
-from .base import Featurizer, S, T, VectorFeaturizer, GraphFeaturizer
-from .atom import MultiHotAtomFeaturizer, AtomFeatureMode, get_multi_hot_atom_featurizer
+from .atom import AtomFeatureMode, MultiHotAtomFeaturizer, get_multi_hot_atom_featurizer
+from .base import Featurizer, GraphFeaturizer, S, T, VectorFeaturizer
```

Comment:
Switching the order of these imports has caused a circular import error. Importing from .atom attempts to import something from chemprop.featurizers.base, which goes back to this file and imports .atom again.

Atom needs to import from .base instead of the full path